api/v1alpha1/alluxioruntime_types.go

```diff
@@ -295,6 +295,10 @@ type AlluxioRuntimeSpec struct {
 	// CleanCachePolicy defines cleanCache Policy
 	// +optional
 	CleanCachePolicy CleanCachePolicy `json:"cleanCachePolicy,omitempty"`
+
+	// SkipMetadataSync defines whether to skip syncing metadata when Alluxio is being set up. Default to false.
+	// +optional
+	SkipMetadataSync bool `json:"skipMetadataSync,omitempty"`
```

Comment:
How about `disableMetadataSync`?

Reply:
Made `metadataSyncBehavior` and `cacheCleanBehavior` sub-properties of `AlluxioRuntime.spec.management`. We shall define interactions between runtime engine and operator in `AlluxioRuntime.spec.management` in future development